api/config/config_local.json

```diff
@@ -98,7 +98,8 @@
     "renewal": {
       "expiryDays": 365,
       "expiryDaysWarningThresholds": [1, 3, 7, 15, 30],
-      "lookbackPeriod": 330
+      "lookbackPeriod": 330,
+      "trainingLookbackPeriod": 30
```

Comment:
I got an idea for future improvement: there is a pattern here in renewal configs. Maybe one of more config values can be derived.
 
TOL, e.g. `trainingLookbackPeriod` can be derived from last element of `expiryDaysWarningThresholds` array. Or `expiryDays` minus `lookbackPeriod` (= 35) is what `trainingLookbackPeriod` should be.